### PR TITLE
[pdmo-71]redesign the names and demographics components

### PR DESCRIPTION
**Pull request by CeciliaWanjiku**
# JIRA TICKET NAME:

PDMO-71: [Redesign the names and demographics component](https://issues.openmrs.org/browse/PDMO-71)
# SUMMARY:

Implement Feedback and change the display of the names & demographics component

### CODE REVIEW

File: app/js/components/names.jsx [hunk 1]
```diff
@@ -134,7 +134,12 @@ class Name extends React.Component {
     }
     return (
       <div>
-        <form className='form-horizontal'>
+          <h3 className=''
+            name=''
+          > {this.state.givenName} {' '} {this.state.middleName } {''} {this.state.familyName}</h3>
+          <a> Edit </a>
+
```
Comment (jacksono) [inline app/js/components/names.jsx]: You have classname and name properties for the h3 tag but not assigned to anything, you could just omit them instead if they serve no purpose.

**Review by jacksono — COMMENTED** [review]
Nice job

**Comment by Ewanjiru** [comment]
Looks good

File: app/js/components/names.jsx [hunk 2]
```diff
@@ -134,7 +134,12 @@ class Name extends React.Component {
...
+        {/*<form className='form-horizontal'>
```
Comment (dkayiwa) [inline app/js/components/names.jsx]: Did you also look at our conventions in regards to commented out code? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

**Comment by CeciliaWanjiku** [comment]
@dkayiwa the commented out code is the logic for edit. Waiting for pdmo-73 to get merged so that i can use the modal to avoid inconsistency issues.

**Comment by dkayiwa** [comment]
@CeciliaWanjiku the author of pdmo-73 has not yet responded to the review comments.

**Pull request closed without merging**